tests/integration_tests/functional/test_api.py

```diff
@@ -393,6 +393,7 @@ def test_api_mmds_config(test_microvm_with_api):
     )['mmds-config']['version'] == "V2"
 
 
+# pylint: disable=too-many-statements
```

Comment:
Maybe we should split this test into multiple parts so that we get rid of this error. The test is indeed pretty long now.

Reply:
It's fairly long but I can't think of a way to purposefully split this test into multiple ones. It's a pretty cohesive one.